apps/tng-dictionary/src/components/Loading/Loading.tsx

```diff
@@ -16,28 +16,14 @@ export function Loading(): JSX.Element {
   return (
     <div className='load' style={background}>
 
-      <div className='loading' style={{
-        backgroundImage: ''
-
-      }}>
+      <div className='loading'>
         <Box sx={{ display: 'flex' }}>
-          <CircularProgress style={styles.loader} thickness={4} size={140} />
+          <CircularProgress style={styles.loader} thickness={4} size={140} variant="indeterminate" />
           <CircularProgress style={styles.track} thickness={4} size={140} value={100} variant="determinate" />
           <CircularProgress style={styles.background} thickness={22} size={140} value={100} variant="determinate" />
-          <motion.div
-            initial={{
-              opacity: 0.5
-            }}
-            animate={{
-              opacity: 1
-            }}
-            exit={{
-              opacity: 0
-            }}
-            transition={{ duration: 1.5 }}
-          >
-            <div className='Logo'><img height={80} src={logo} /></div>
-          </motion.div>
+          <div className='Logo'>
+            <img height={80} src={logo} />
```

Comment:
We should make this configurable so that other groups can use their logo as well.